### ISSUE TITLE
Errors with keyboard control

### ISSUE THREAD

**Issue by trollixx** [fnagel/jquery-ui#122]
I noticed strange behaviour of selectmenu, when I try to control it with keyboard.
1. Focus on select menu. Press Space. Select any item with arrow keys. Press Escape. If you open dropdown again - last selected item will be highlighted.
2. Using arrow key on closed selectmenu do not make selected item be selected in dropdown.

Tested on:
Chromium 13.0.767.1 (Сборка для разработчиков 85531) Ubuntu 11.04

**Comment by fnagel** [comment]
1. This is the correct behaviour as the last focused state should be focused. The base style of jQuery UI is somewhat imperfect in this specific case. try using the themeswitcher.
2. The last focused option is not reseted when changing the selected element via closed menu with arrow keys. I aggree, this should be changed, but Im not quite sure how to do this. Any suggestions?

**Comment by trollixx** [comment]
Can we move focus on dropdown opening?

**Comment by fnagel** [comment]
Steps to reproduce: use keyboard, open menu, select item, close, change item in closed state, open again

No, this would remove focus in any circumstance. 

Ive did some changes within the latest commit, should be fixed now.
Please test and give feedback!

**Comment by fnagel** [comment]
Please give some feedback!

**Comment by fnagel** [comment]
Any feedback on this issue?

**Comment by trollixx** [comment]
Sorry, was busy with other projects, will try test at the weekend.

**Comment by fnagel** [comment]
_ping_

**Comment by trollixx** [comment]
Sorry again. Just tested it, works pretty well in expected way. So, the 2nd problem of this issue is fixed and the 1st one is jQuery UI's 'feature'.

**Comment by fnagel** [comment]
Nice, thanks for your feedback!
